datafusion/src/physical_plan/functions.rs

```diff
@@ -512,39 +512,35 @@ macro_rules! invoke_if_unicode_expressions_feature_flag {
     };
 }
 
-/// Create a physical (function) expression.
-/// This function errors when `args`' can't be coerced to a valid argument type of the function.
-pub fn create_physical_expr(
+/// Create a physical scalar function.
```

Comment:
this change has been split out into https://github.com/apache/arrow-datafusion/pull/548 for easier review.